tests/Unit/Functions/Options/rocketDeleteLicenceDataFile.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace WP_Rocket\Tests\Unit\Functions\Options;
+
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+use org\bovigo\vfs\vfsStream,
+	org\bovigo\vfs\vfsStreamDirectory;
+
+/**
+ * @covers rocket_delete_licence_data_file
+ * @group Functions
+ * @group Options
+ */
+class Test_RocketDeleteLicenceDataFile extends TestCase {
+	private $path;
+	private $mock_fs;
+
+	public function setUp() {
+		parent::setUp();
+
+		require WP_ROCKET_PLUGIN_ROOT . 'inc/functions/options.php';
```

Comment:
`require_once` instead. Why? `setUp()` runs for each test method in the class. Right now, there's only 1 test. Later we may have more. What happens if `setUp()` runs more than once? Fatal error. Switching it to `require_once` resolves that problem.